Makefile

```
@@ -1,12 +1,20 @@
+BAKE_OUT=frontend/public/baked
+ASSETS_IN=assets/3d
+# Allow overriding the output directory via `make BAKE=/tmp/output`
+BAKE?=$(BAKE_OUT)
 
-tf\:init:
-terraform -chdir=ops/terraform init
+# bake: convert 3D source assets (*.3ds/*.obj/*.fbx) in $(ASSETS_IN)
+# into GLB files written to $(BAKE). Requires the assimp CLI to be
+# installed locally. Example: `make bake ASSETS_IN=assets/custom BAKE=build/glbs`.
+# The target fails fast if assimp is unavailable or a conversion errors.
+bake:
```

checkmake (0.2.2)

[warning] Target body for "bake" exceeds allowed length of 5 (8).

(maxbodylength)